packages/table/src/Table.tsx

```diff
@@ -14,6 +19,42 @@ const tableStyles = css`
   border-bottom: 1px solid ${uiColors.gray.light2};
 `;
 
+const ScrollState = {
+  None: 'none',
+  Left: 'left',
+  Right: 'right',
+  Both: 'both',
+} as const;
+
+type ScrollState = typeof ScrollState[keyof typeof ScrollState];
+
+function getScrollShadowStyle(scrollState: ScrollState): string {
+  const shadowColor = 'rgba(93,108,116,0.3)';
+
+  if (scrollState === ScrollState.Both) {
```

Comment:
can we use a `switch` and explicitly handle `ScrollState.None`? if another scroll state were added in the future we'd want this function to have a type error

Reply:
Yes makes sense, but for now going to write

```ScrollState.None: 
default:
```

because the default should be handled the same way

Reply:
> because the default should be handled the same way

There is no default case because the only cases possible are those enumerated by `ScrollState`.

If we omit `default:` Typescript won't complain now because all possible cases of `ScrollState` have been handled, but we want it to complain when adding new cases to the enum.

Reply:
Removed!